packages/ai/src/openai/azure.ts

```diff
@@ -405,10 +424,9 @@ export class WrappedResponses extends AzureOpenAI.Responses {
               await sendEventToPosthog({
                 client: this.phClient,
                 ...posthogParams,
-                //@ts-expect-error
-                model: openAIParams.model,
+                model: openAIParams.model ?? '',
```

Comment:
Could we not send an empty string instead? Didn't check if it makes sense though.